webapp/sdf/src/components/item-component/item-component.spec.js

```diff
@@ -0,0 +1,61 @@
+import { mount } from '@vue/test-utils'
+import ItemComponent from 'components/item-component/item-component.vue'
+import Item from 'classes/item.js'
+
+describe('item-component', () => {
+let component
+
+    beforeEach(() => {
+        component = mount(ItemComponent, {
+            propsData: {
+                item : new Item(1, "Eintrag2", 2)
+            }
+        });
+    })
+
+    it('contains item title and votes Expected --> Eintrag (2)', () => {
+        expect(component.text()).toContain('Eintrag2 (2)')
+    })
+
+    it('emits remove event', async () => {
+        const removeButton = component.find('.remove-button')
+        removeButton.trigger('click')
+        await component.vm.$nextTick()
+        expect(component.emitted().removeEvent).toBeTruthy()
```

Comment:
```suggestion
        expect(component.emitted('removeEvent')).toBeTruthy()
```
https://vue-test-utils.vuejs.org/api/wrapper/emitted.html#emitted